.happy/terraform/modules/sfn/main.tf

```diff
@@ -174,6 +174,12 @@ resource "aws_sfn_state_machine" "state_machine" {
         "Type": "Task",
         "InputPath": "$",
         "Resource": "${var.lambda_error_handler}",
+        "Parameters": {
+          "execution.$": "$$.Execution.Id",
```

Comment:
I would change the name of the key to `"execution_id.$"` since it is the id of the execution object/block